src/mocking/handlers.ts

```diff
@@ -113,6 +120,18 @@ const queryNodeHandlers = [
     createCursorPaginationAccessor<GetChannelsConnectionQuery['channelsConnection']>(mockChannels)
   ),
 
+  // memberships
+  createQueryHandler<GetMembershipQuery, GetMembershipQueryVariables>(
+    queryNode,
+    GetMembershipDocument,
+    createSingleItemAccessor(mockMemberships)
+  ),
+  createQueryHandler<GetMembershipsQuery, GetMembershipsQueryVariables>(
+    queryNode,
+    GetMembershipsDocument,
+    createMembershipsAccessor(mockMemberships)
```

Comment:
Do we need this custom accessor? I'd think the regular `createOffsetLimitPaginationAccessor` would work here?